docs/source/distributions/self_hosted_distro/fireworks.md

```diff
@@ -18,7 +18,7 @@ The `llamastack/distribution-fireworks` distribution consists of the following p
 | agents | `inline::meta-reference` |
 | datasetio | `remote::huggingface`, `inline::localfs` |
 | eval | `inline::meta-reference` |
-| inference | `remote::fireworks` |
+| inference | `remote::fireworks`, `inline::sentence-transformers` |
```

Comment:
doesnt fireworks have an embedding model?

Reply:
@raghotham yes, it does but we use `all-miniLM` hard coded in all our tests 🤡

Reply:
I started removing this and realized I cannot because of this limitation. Ollama worked out because it actually has the same exact model available :)